src/lib/utils/index.test.ts

```diff
@@ -1,49 +1,63 @@
 import { BigNumber } from '@ethersproject/bignumber'
 
-import { displayFromWei, formatDollarAmount, isValidTokenInput, toWei } from '.'
+import {
+  formatAmountFromWei,
+  formatDollarAmount,
+  isValidTokenInput,
+  toWei,
+} from '.'
 
-describe('displayFromWei', () => {
-  it('should return null with no number provided', () => {
-    const displayValue = displayFromWei(undefined)
-    expect(displayValue).toBe(null)
-  })
-  it('should convert to token value, default full value', () => {
-    const displayValue = displayFromWei(BigNumber.from('180313000000000000000'))
+describe('formatAmountFromWei', () => {
+  it('should return formatted amount for given wei', () => {
+    const displayValue = formatAmountFromWei(
+      BigInt('180313000000000000000'),
+      18,
+      3,
+    )
     expect(displayValue).toBe('180.313')
-    const displayValue2 = displayFromWei(BigNumber.from('1'))
+    const displayValue2 = formatAmountFromWei(BigInt('1'), 18, 18)
```

Comment:
Adapting the tests, I noticed there won't be such thing as leaving everything default or passing a null/undefined value. But I guess in the end this is more precise/ok?

Reply:
I like this approach/being explicit here 💯